Uplift/Controllers/OnboardingGymsViewController.swift

```diff
@@ -33,31 +33,50 @@ class OnboardingGymsViewController: UIViewController, UITableViewDelegate, UITab
     // Gyms
     private var gymNames: [String] = []
     private var favoriteGyms: [String] = []
-    
+
     weak var delegate: ChooseGymsDelegate?
 
     override func viewDidLoad() {
         super.viewDidLoad()
         view.backgroundColor = .primaryWhite
         let defaults = UserDefaults.standard
-        let buttonBorderSize: CGFloat = 2
         
         // Get Favorite Gyms from UserDefaults
         if let favorites = defaults.array(forKey: Identifiers.favoriteGyms) as? [String] {
             favoriteGyms = favorites
         }
-        
+
         // Set up screen sizes for scaling
         currentScreenSize = computeScreenDimensions()
 
+        NetworkManager.shared.getGymNames { gyms in
+            var namesArray = [String]()
+            gyms.forEach { namesArray.append($0.name) }
```

Comment:
you shouldn't need this anymore right?

Reply:
oops poorly named endpoint/query.... `getGymNames` gets names + ids

Reply:
it's used elsewhere so i can't change it to just names. should i change the endpoint & query name? (getGymNamesAndIds....?)